readme.md

```diff
@@ -1,14 +1,44 @@
-# oneleif Active Projects
-* [NoteMapV2](https://github.com/zmeriksen/NoteMapV2)
-* [kyy](https://github.com/parshav/kyy)
-* [ZombieMixer](https://github.com/ambid17/ZombieMixer)
-* [SwiftObject](https://gitlab.com/_leif/SwiftObject)
+# oneleif documents
 
-## CocoaPods
-* [SwiftObject](http://cocoapods.org/pods/SwiftObject)
 
-## Javascript
-* [ViaJS](https://github.com/abdi0987/ViaJS)
+### [Information](https://github.com/oneleif/olDocs/tree/master/Information)
 
-## oneleif Discord Link
-[Discord](https://discord.gg/wkekGe3)
+### [Code Standards](https://github.com/oneleif/olDocs/tree/master/CodeStandards)
+
+### [Markdown Components](https://github.com/oneleif/olDocs/tree/master/MarkdownComponents)
+
+### [Assets](https://github.com/oneleif/olDocs/tree/master/assets)
+
+## Open Resources
+
+- [3D Models](https://github.com/oneleif/open-3d-models)
+- [2D Assets](https://github.com/oneleif/open-2d-assets)
```

Comment:
```suggestion
### [3D Models](https://github.com/oneleif/open-3d-models)
### [2D Assets](https://github.com/oneleif/open-2d-assets)
```

These should be styled consistently with the above documents

Reply:
I'm on the fence about this. I agree with consistency, but these links are different from the ones above. These links are to other GitHub repos and will take them away from this repo. The other links above are all contained inside this repo. 

What do you think? I'm fine changing them, but does that make sense that they look the same, but act somewhat different?